yt/yt/tests/integration/scheduler/test_get_job_artifacts.py

```diff
@@ -776,6 +776,75 @@ def do_test_get_job_stderr(self):
     def test_get_job_stderr(self):
         self.do_test_get_job_stderr()
 
+    def do_test_get_job_stderr_limit_offset(self):
+        create("table", "//tmp/t1")
+        create("table", "//tmp/t2")
+        write_table("//tmp/t1", [{"foo": "bar"}, {"foo": "baz"}, {"foo": "qux"}])
+
+        op = map(
+            track=False,
+            label="get_job_stderr",
+            in_="//tmp/t1",
+            out="//tmp/t2",
+            command=with_breakpoint(
+                "for (( i=0; i<10; i+=1 )); do echo current cycle is $i >&2; sleep 0.1; done ; echo STDERR-FINISH >&2 ; BREAKPOINT ; cat"
+            ),
+            spec={"mapper": {"input_format": "json", "output_format": "json", "max_stderr_size": 1000}},
+        )
+
+        job_id = wait_breakpoint()[0]
+
+        wait(
+            lambda: retry(lambda: get_job_stderr_paged(op.id, job_id, limit=0, offset=-50))["data"].endswith(
+                b"STDERR-FINISH\n"
+            )
+        )
+        release_breakpoint()
+        op.track()
```

Comment:
As far as I understand job is complete here and stderr is received from job archive. Is it really our intention here?